model/src/main/proto/spine_examples/shareaware/share/share.proto

```diff
@@ -26,12 +26,12 @@
 
 syntax = "proto3";
 
-package spine_examples.shareaware;
+package spine_examples.shareaware.share;
```

Comment:
I have asked to adjust the Java package only.

Let's leave the Proto message in the same folder and in the same Proto package.